src/console_reporter.cc

```diff
@@ -150,7 +150,9 @@ void ConsoleReporter::PrintRunData(const Run& result) {
   for (auto& c : result.counters) {
     const std::size_t cNameLen = std::max(std::string::size_type(10),
                                           c.first.length());
-    auto const& s = HumanReadableNumber(c.second.value, 1000);
+    const double one_k =
+        double(c.second.thousand.first) / double(c.second.thousand.second);
```

Comment:
why is this specified as a pair instead of just a single number (1000 vs 1024)? or even an enum!

Reply:
Ah, I think that is suggesting either having an enum with values 1000 and 1024, or to extend Counter to have MegaCounter and MebiCounter variants, which implicitly manage the one_k definition without it being specified.

Reply:
Right now i don't have/is not aware of any real use for the one_k not being 1000 or 1024, so enum _could_ work.
It is just that storing enum will take just as much space as storing an int, or a pair of shorts:
https://godbolt.org/z/9Q1gVS
So it isn't obvious to me why we want to avoid doing/allowing the more general thing at the **same** cost.